Testing - packagingv2

Salamandar opened the pull request:
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

yunohost-bot committed:
Auto-update README

Salamandar committed:
Manifest v2

yunohost-bot committed:
Auto-update READMEs

Salamandar committed:
full_replace=1

Salamandar committed:
Fix tests

Salamandar committed:
Fix variable name

Salamandar committed:
Fix scripts/actions

yunohost-bot committed:
Auto-update READMEs

Salamandar committed:
Fix test_upgrade_from

Salamandar committed:
Fix post_install message

Salamandar committed:
Change version of test_upgrade_from

Salamandar committed:
Fix linter, remove fixmes

Salamandar committed:
Merge pull request #61 from YunoHost-Apps/manifestv2

Manifestv2

Salamandar commented:
!testme

Salamandar committed:
Fix frequencies settings during upgrade

alexAubin committed:
Delete actions.toml: this doesn't do anything since quite a long time ago ...

yunohost-bot committed:
Auto-update READMEs

alexAubin commented on doc/POST_INSTALL.md:
```diff
@@ -0,0 +1,14 @@
+If encryption is enabled, your password for encryption is '__ENCRYPTION_PWD__'
```
Hmpf displaying user-provided password in cleartext in an UI, idk how i feel about that x_x

alexAubin commented on doc/POST_INSTALL.md:
```diff
@@ -0,0 +1,14 @@
+
+Please read the [documentation](https://github.com/maniackcrudelis/archivist/blob/master/Configuration.md) about the configuration of archivist for more information.
+
+You can configure this app easily by using the experimental [config-panel feature](https://__DOMAIN__/yunohost/admin/#/apps/__APP__/config-panel).  
+You can also find some specific actions for this app by using the experimental [action feature](https://__DOMAIN__/yunohost/admin/#/apps/__APP__/actions).
+
+If you're facing an issue or want to improve this app, please open a new issue in this [project](https://github.com/YunoHost-Apps/archivist_ynh).
```
```suggestion
```

Actions don't exist anymore, config panel is not experimental anymore, and there should already be a link to the bugtracker in the same page this stuff is displayed

Salamandar replied:
Thanks :) TBH i only copa pasta-ed the text.

alexAubin commented on scripts/actions/clean_backups:
```diff
@@ -1,39 +1,12 @@
 #!/bin/bash
```
This file doesn't do anything anymore

alexAubin commented on scripts/actions/force_backup:
```diff
@@ -1,45 +1,18 @@
 #!/bin/bash
```
This file doesn't do anything anymore

alexAubin commented on scripts/actions/reset_default_app:
```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
```
This file doesn't do anything anymore

alexAubin commented on scripts/actions/reset_default_config:
```diff
@@ -1,44 +1,22 @@
 #!/bin/bash
```
This file doesn't do anything anymore

alexAubin committed:
Delete YEP.md

alexAubin committed:
Delete CHANGELOG.md

Salamandar committed:
Update doc/POST_INSTALL.md

Actions don't exist anymore, config panel is not experimental anymore, and there should already be a link to the bugtracker in the same page this stuff is displayed

Co-authored-by: Alexandre Aubin <4533074+alexAubin@users.noreply.github.com>

yunohost-bot committed:
Auto-update READMEs

alexAubin committed:
Delete old actions/ folder

ericgaspar commented:
!testme

lapineige commented:
:up:
